tests/python/opinfo_input_generators.py

```diff
@@ -1523,13 +1523,13 @@ def linear_input_generator(
 
     # Cases without bias
     shapes_input = ((K), (M, K), (B, M, K), (B, 1, M, K))
-    shapes_weight = ((K), (N, K), (1, K))
+    shapes_weight = ((N, K), (1, K))
     for shape_input, shape_weight in itertools.product(shapes_input, shapes_weight):
         yield SampleInput(make_arg(shape_input), make_arg(shape_weight))
 
     # Cases with bias
     shape_weight = (N, K)
-    shapes_bias = ((), (N,))
```

Comment:
Did this PR break no bias support?

Reply:
No, it only removes 0-D bias support. No-bias support is tested a few lines above.